src/main/java/service/Parser.java

```diff
@@ -0,0 +1,7 @@
+package service;
+
+import model.TransactionDto;
+
+public interface Parser {
+    TransactionDto parseLine(String line);
```

Comment:
```suggestion
    List<Transaction> parse(List<String> lines);
```
Let`s remove the loop from main() method